src/main/java/com/google/cloud/tools/gradle/appengine/task/io/GradleLoggerOutputListener.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2016 Google Inc. All Right Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.google.cloud.tools.gradle.appengine.task.io;
+
+import com.google.cloud.tools.appengine.cloudsdk.process.ProcessOutputLineListener;
+
+import org.gradle.api.logging.LogLevel;
+import org.gradle.api.logging.Logger;
+
+/**
+ * Output listener that writes to a gradle logger
+ */
+public class GradleLoggerOutputListener implements ProcessOutputLineListener {
```

Comment:
It's actually necessary for output capture it turns out. So there was no way to read the output unless we're redirecting to the gradle logger. The inherited redirect made all the tests fail.